.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 
 script:
     - if [[ $TEST_TARGET == 'default' ]]; then
-        python setup.py test -a "--verbose";
+        python setup.py test -a '--verbose --cov=pyugrid';
```

Comment:
The report is at the end of the log. We can add a badge using those
coverage services, but I don't really like those.
On Feb 18, 2016 1:24 PM, "Chris Barker" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/pyugrid/pyugrid/pull/130#discussion_r53338782>:
>
> > @@ -26,7 +26,7 @@ install:
> >
> >  script:
> >      - if [[ $TEST_TARGET == 'default' ]]; then
> > -        python setup.py test -a "--verbose";
> > +        python setup.py test -a '--verbose --cov=pyugrid';
>
> so what happens if you don't have 100% coverage? I'd like to see the
> report, but we don't want that to be the equivalent of a test failure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyugrid/pyugrid/pull/130/files#r53338782>.
>